certs/cert.pem

```diff
@@ -0,0 +1,34 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
@petorsong not sure if this file should be committed - it was already in the git history lol

Can you confirm if I should or shouldn't remove this?